pyproject.toml

```
@@ -11,14 +11,18 @@ repository = "https://github.com/qdrant/fastembed"
 keywords = ["vector", "embedding", "neural", "search", "qdrant", "sentence-transformers"]
 
 [tool.poetry.dependencies]
-python = ">=3.8.0,<3.12"
-onnx = "^1.11"
-onnxruntime = "^1.15"
-tqdm = "^4.65"
+python = ">=3.8.0,<3.13"
+onnx = "^1.15.0"
+onnxruntime = "^1.17.0"
+tqdm = "^4.66"
 requests = "^2.31"
-tokenizers = "^0.15.0"
-huggingface-hub = "0.19.4"
+tokenizers = "^0.15.1"
+huggingface-hub = "^0.20.3"
```

```suggestion
huggingface-hub = "^0.20"
```

Relax version lock for Huggingface Hub